OneSignalSDK/unittest/src/test/java/com/test/onesignal/MainOneSignalClassRunner.java

```diff
@@ -2753,7 +2779,9 @@ public void testSessionTimeTrackingOnPrivacyConsent() throws Exception {
 
       // Provide consent to OneSignal SDK amd forward time 2 minutes
       OneSignal.provideUserConsent(true);
-      threadAndTaskWait();
+      // TODO: This passed without a resume here, why?
```

Comment:
Did you mean why doesn't this pass without a resume? `OneSignalInit();` already calls resume.

Reply:
Yes, sorry. This "doesn't" pass without resume here for some reason